### PR TITLE
[don't merge] Rebase v5.x.x on top of v4.x.x

### PR DESCRIPTION
**Pull request by nemanja-boric-sociomantic**
CI test.

### CODE REVIEW

File: src/swarm/neo/client/RequestSet.d [hunk 1]
```diff
@@ -351,18 +315,17 @@ public final class RequestSet: IRequestSet
                     can be packed by swarm.neo.util.StructPacker
                 handler = request handler
                 finished_notifier = called when the last hander has finished
-                context = request context
-                working = initial working data to serialize into the request
-                    connections
```
Comment (gavin-norman-sociomantic) [inline src/swarm/neo/client/RequestSet.d]: The changes from
```
context = request context
```
to
```
context = request context, must be a Contiguous-serialisable
    struct
```
are wrong. See 9c87a4682854d52c8569993073343e9127569f03.

**Review by gavin-norman-sociomantic — COMMENTED** [review]
One small merge error in docs...

File: src/swarm/neo/client/RequestSet.d (hunk 1 above)
Comment (nemanja-boric-sociomantic) [reply src/swarm/neo/client/RequestSet.d]: Oh, yes!

**Comment by nemanja-boric-sociomantic** [comment]
Updated with the doc fix.

**Review by gavin-norman-sociomantic — APPROVED** [review]
Please push at will.

**Comment by nemanja-boric-sociomantic** [comment]
Pushed.

**Pull request closed without merging**